tests/h2-tests/tests/stream_states.rs

```diff
@@ -829,7 +753,7 @@ fn rst_while_closing() {
         .recv_settings()
         .recv_frame(
             frames::headers(1)
-                .request("GET", "https://example.com/")
+                .request("POST", "https://example.com/")
```

Comment:
Is the change from GET to POST here related to the rest of this change?

Reply:
I was searching for GET, and then part way into replacing this, realized it wants the `SendStream`, so POST seemed clearer.